eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -655,6 +669,57 @@ private function prependTranslatorConfiguration(ContainerBuilder $container)
         }
     }
 
+    private function prependDoctrineConfiguration(ContainerBuilder $container): void
+    {
+        if (!$container->hasExtension('doctrine')) {
+            return;
+        }
+
+        $processedDoctrineConfig = $this->processConfiguration(
+            new DoctrineConfiguration(false),
+            $container->getExtensionConfig('doctrine')
+        );
+
+        $kernelConfigs = $container->getExtensionConfig('ezpublish');
+        $entityMappings = [];
+
+        $repositoryConnections = [];
+        foreach ($kernelConfigs as $config) {
+            if (isset($config['orm']['entity_mappings'])) {
+                $entityMappings[] = $config['orm']['entity_mappings'];
+            }
+
+            if (isset($config['repositories'])) {
+                $repositoryConnections[] = array_map(
+                    static function (array $repository) use ($processedDoctrineConfig): ?string {
+                        return $repository['storage']['connection'] ?? $processedDoctrineConfig['dbal']['default_connection'] ?? 'default';
```

Comment:
CS - line too long, unreadable
Perhaps:
```suggestion
                        return $repository['storage']['connection']
                            ?? $processedDoctrineConfig['dbal']['default_connection']
                            ?? 'default';
```